package.json

```diff
@@ -1,32 +1,42 @@
 {
-    "name": "camelize2",
-    "version": "1.0.0",
-    "description": "recursively transform key strings to camel-case",
-    "main": "index.js",
-    "devDependencies": {
-        "mocha": "6.1.2"
-    },
-    "scripts": {
-        "test": "mocha test/*.js"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git://github.com/DenysIvko/camelize2.git"
-    },
-    "homepage": "https://github.com/DenysIvko/camelize2",
-    "keywords": [
-        "camel-case",
-        "json",
-        "http",
-        "response",
-        "transform"
-    ],
-    "author": {
-        "name": "Denys Ivko",
-        "email": "ivkodenis92@gmail.com"
-    },
-    "license": "MIT",
-    "dependencies": {
-        "camelcase": "5.3.1"
-    }
+  "name": "camelize2",
+  "version": "1.0.1",
```

Comment:
Would you mind to change bump MINOR instead of PATCH? `1.0.1` => `1.1.0`
according to https://semver.org/ :
MINOR version when you add functionality in a backwards compatible manner,